UI.py

```diff
@@ -193,6 +193,33 @@ def CreateLeavesBalancesEmbed(member_id):
         print(e)
         return None
 
+def CreateInformMemberOfLeaveStatusEmbed(status, admin_name, reason, leave_type, start_date, end_date):
+    leaveImages = {
+    "Annual"   : os.getenv("Annual_Leave_Link"),
+    "Sick"     : os.getenv("Sick_Leave_Link"),
+    }
+    embed = discord.Embed(
+        title = f'Leave Request Status',
+        description = f'Your request was ' + status.lower(),
```

Comment:
i think stating the status in the very beginning of the embed, makes the `embed.add_field(name = "Status", value = status, inline = True)` obsolete.

```suggestion
        description = f'Request No. ' + request_id,
```
would be better, no ? @AntWong-TheBadNomads

Reply:
I agree. Switched to display request id.